src/components/wells/Sample.vue

```diff
@@ -83,7 +84,8 @@ export default {
 
       const qcAssay = this.store.qcAssayList.find(this.plateBarcode)
       // pull out settings defined in the quantTypes.json config
-      if (qcAssay && qcAssay.quantType && qcAssay.quantType.qcResults) {
+      // if we get more requests for this type of warning worth adding the quantType to the store
+      if (qcAssay?.quantType?.qcResults) {
```

Comment:
I'm glad this works here. In traction-ui I've had compile errors if I attempt to do this in a vue component. (I'm assuming its babel not quite being set up right)